<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qtomato</class>
 <widget class="QDialog" name="Qtomato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qtomato</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="buttonPomodoro">
     <property name="styleSheet">
      <string notr="true">QPushButton { color: #ffffff; background-color: #337ab7; }</string>
     </property>
     <property name="text">
      <string>Pomodoro</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonShortBreak">
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #5bc0de; }</string>
     </property>
     <property name="text">
      <string>Short Break</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="buttonLongBreak">
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #5bc0de; }</string>
     </property>
     <property name="text">
      <string>Long Break</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="buttonStart">
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #5cb85c; }</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonStop">
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #d43f3a; }</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="buttonReset">
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #f0ad4e; }</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="time">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Mono</family>
       <pointsize>72</pointsize>
      </font>
     </property>
     <property name="text">
      <string>25:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
